open-metadata-implementation/access-services/glossary-view/glossary-view-api/src/main/java/org/odpi/openmetadata/accessservices/glossaryview/rest/GlossaryViewEntityDetailFactory.java

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.glossaryview.rest;
+
+public class GlossaryViewEntityDetailFactory {
+
+    private final static String GLOSSARY = "Glossary";
+    private final static String CATEGORY = "GlossaryCategory";
+    private final static String TERM = "GlossaryTerm";
+
+    public static GlossaryViewEntityDetail build(String entityType){
+        GlossaryViewEntityDetail prototype = null;
+        if(entityType.equalsIgnoreCase(GLOSSARY)){
```

Comment:
Please check this for the next PR. 
It dose same thing for all conditions.
Add else in order not all conditions to be checked.